app/assets/javascripts/views/dashboard.js

```diff
@@ -98,7 +98,7 @@
     // sense to let the session expire
 
     $.ajax({
-      url: self.params.url,
+      url: ELMO.app.url_builder.build('/'),
```

Comment:
I honestly don't remember changing this :) Perhaps this got upgraded somehow automatically?